### PR TITLE
LSO - Mantis 25075 - rename start btn in kiosk mode

### PR DESCRIPTION
**Pull request by daniwe4**

### CODE REVIEW

File: Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php [hunk 1]
```diff
@@ -28,17 +28,14 @@ public function __construct(
 		$this->access = $access;
 	}
 
-	/**
-	 * @inheritDoc
-	 */
-	protected function getObjectClass(): string
+	protected function getObjectTitle(): string
```
Comment (klees) [inline Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php:31]: Do we still need this method?

File: Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php [hunk 2]
```diff
@@ -227,4 +224,42 @@ private function debugBuildAllControls(ControlBuilder $builder): ControlBuilder
...
+				break;
+			case "svy":
+				$title = $this->lng->txt("obj_svy");
+				break;
+			default:
+				$title = "";
+				break;
+		}
+
+		return $title;
```
Comment (klees) [inline Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php]: Why not just use `return $this->lng->txt("obj_$type");` here? This IMO would work for plugins too.

**Review by klees — CHANGES_REQUESTED** [review]
Hi @daniwe4. This looks as if it would work for most object-types, but could be a lot simpler.

File: Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php (hunk 1 above)
Comment (daniwe4) [reply Modules/LearningSequence/classes/Player/class.ilLegacyKioskModeView.php:31]: The method 'debugBuildAllControls' use this method.

**Comment by klees** [comment]
Cherry-picked to release_5-4 as well.

**Pull request merged**